app/src/main/java/ru/androidacademy/gooseeye/viewmodels/MovieDetailsViewModel.kt

```diff
@@ -0,0 +1,23 @@
+package ru.androidacademy.gooseeye.viewmodels
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import ru.androidacademy.gooseeye.data.Actor
+import ru.androidacademy.gooseeye.data.Movie
+
+class MovieDetailsViewModel(private val movie: Movie) : ViewModel() {
+
+    private val _movieDetails = MutableLiveData<Movie>()
+    val movieDetails get() = _movieDetails
+    private val _actors = MutableLiveData<List<Actor>>()
+    val actors get() = _actors
+
+    init {
+        getMovie()
+    }
+
+    private fun getMovie() {
```

Comment:
when a function is called `get*` it means that it should return something.
In your case it is something like `loadMovies`